styles/globals.css

```diff
@@ -53,45 +68,105 @@ h1, h2, h3, p, a {
 	-moz-animation: blink .75s linear infinite;
 	-ms-animation: blink .75s linear infinite;
 	-o-animation: blink .75s linear infinite;
-	 animation: blink .75s linear infinite;
+	animation: blink .75s linear infinite;
 }
+
 @-webkit-keyframes blink {
-	0% { opacity: 1; }
-	50% { opacity: 1; }
-	50.01% { opacity: 0; }
-	100% { opacity: 0; }
+	0% {
+		opacity: 1;
+	}
+
+	50% {
+		opacity: 1;
+	}
+
+	50.01% {
+		opacity: 0;
+	}
+
+	100% {
+		opacity: 0;
+	}
 }
+
 @-moz-keyframes blink {
-	0% { opacity: 1; }
-	50% { opacity: 1; }
-	50.01% { opacity: 0; }
-	100% { opacity: 0; }
+	0% {
```

Comment:
For just one attribute, I personally like having it all on one line, it’s a little easier to read one line and get all the info instead of breaking it up over 3.

but I also don’t care that much. Keep the PRs comin!